### PR TITLE
Initial PR including files + fwd templates.

### PR DESCRIPTION
**Pull request by carlbastos**

### CODE REVIEW

File: fwd-template.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "output": ".",
+    "builds": [
+        {
+            "name": "default",
```
Comment (dbpolito) [inline fwd-template.json]:
```suggestion
            "name": "7.4-fpm-alpine",
```

Comment (dbpolito) [reply fwd-template.json]: Would be good if we add `7.1, 7.2, 7.3`

File: template/Dockerfile.blade.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    PHP_UPLOAD_MAX_FILESIZE=50M \
+    PHP_POST_MAX_SIZE=50M \
+    PHP_MAX_EXECUTION_TIME=300
+
+ARG DOCKERIZE_VERSION=v0.6.1
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+COPY kool.ini /kool/kool.tmpl
```
Comment (dbpolito) [inline template/Dockerfile.blade.php:14]: Add `cp "/usr/local/etc/php/php.ini-production" "/usr/local/etc/php/php.ini"`

File: template/Dockerfile.blade.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
+FROM {{ $from }}
+
+ENV PHP_MEMORY_LIMIT=256M \
+    PHP_UPLOAD_MAX_FILESIZE=50M \
+    PHP_POST_MAX_SIZE=50M \
+    PHP_MAX_EXECUTION_TIME=300
```
Comment (dbpolito) [inline template/Dockerfile.blade.php]: max execution `30`

File: template/Dockerfile.blade.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+ARG DOCKERIZE_VERSION=v0.6.1
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+COPY kool.ini /kool/kool.tmpl
+
+ENTRYPOINT [ "dockerize", "-template", "/kool/kool.tmpl:/usr/local/etc/php/conf.d/kool.ini", "docker-entrypoint.sh" ]
+
+CMD ["php-fpm"]
```
Comment (dbpolito) [inline template/Dockerfile.blade.php]: remove if it's the same form base

File: .github/docker.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@master
+
+    - name: Build image
+      run: docker build --pull -t kooldev/wordpress:${{ matrix.version }} ${{ matrix.version }}
+
+    - name: Test docker images
+      run: |
+        docker run kooldev/wordpress:${{ matrix.version }} php /usr/src/wordpress/index.php
```
Comment (dbpolito) [inline .github/docker.yml]: move up

**Comment by carlbastos** [comment]
@dbpolito I just submitted the CR corrections/suggestions as per your comments. Cheers!

**Review by dbpolito — APPROVED** [review]

**Comment by dbpolito** [comment]
Beautiful ❤️

**Pull request merged**